runtime/compiler/runtime/J9Profiler.cpp

```diff
@@ -1931,6 +1931,39 @@ TR_BlockFrequencyInfo::getRawCount(TR::ResolvedMethodSymbol *resolvedMethod, TR_
    return frequency;
    }
 
+/**   \brief Returns block number of the original block in which current byteCodeInfo was created.
+ *    \details
+ *          It finds the byte code info of the start of the block in which passed bci was originally
+ *          created and returns the block number of that block from the stored information.
+ *          It is useful when we are trying to associate the profiling information with the node which might
+ *          have moved to different blocks with different bci
+ *    \param bci TR_ByteCodeInfo for which original block number is searched for
+ *    \param comp Current compilation object
+ *    \return block number of the original block bci belongs to.
+ *            WARNING: If consumer of this API uses this to to get the profiled data in later compilation and
+ *            requested BCI was not inlined before, it returns -1.
+ */
+int32_t
+TR_BlockFrequencyInfo::getOriginalBlockNumberToGetRawCount(TR_ByteCodeInfo &bci, TR::Compilation *comp)
```

Comment:
I don't think we want to expose this detail - the APIs that need it may belong here instead.

Reply:
Made this Private in https://github.com/eclipse/openj9/pull/2348/commits/cfd65b3bf32bcc9a412d70c12be24218148e1279